lambda/dynamo-api/src/handlers/put-user.ts

```diff
@@ -0,0 +1,32 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
```

Comment:
put-service and put-user files look mostly the same. Any way to reuse the code or better to keep them separate due to use in lambda?